engine/src/main/java/com/datatorrent/stram/StramLocalCluster.java

```diff
@@ -318,6 +329,18 @@ public StramLocalCluster(LogicalPlan dag) throws IOException, ClassNotFoundExcep
     }
   }
 
+  private void addToClasspath(String jar)
```

Comment:
This method mixes two different though related to each other functionalities. One is getting URLClassLoader and modifying "addURL" access that can be done only once and the second is adding Jar URL to the classpath.